waspc/src/Wasp/Psl/Generator/ConfigBlock.hs

```diff
@@ -0,0 +1,28 @@
+module Wasp.Psl.Generator.ConfigBlock
+  ( generateConfigBlockKeyValues,
+    overrideConfigBlockValues,
+  )
+where
+
+import Data.Function (on)
+import Data.Functor ((<&>))
+import Data.List (nubBy)
+import qualified Wasp.Psl.Ast.ConfigBlock as Psl.ConfigBlock
+
+generateConfigBlockKeyValues :: [Psl.ConfigBlock.ConfigBlockKeyValue] -> String
+generateConfigBlockKeyValues keyValues = unlines . map ("  " ++) $ generateConfigBlockKeyValue <$> keyValues
+  where
+    generateConfigBlockKeyValue (Psl.ConfigBlock.ConfigBlockKeyValue key value) = key ++ " = " ++ value
+
+overrideConfigBlockValues :: [(String, String)] -> Psl.ConfigBlock.ConfigBlock -> Psl.ConfigBlock.ConfigBlock
```

Comment:
Ok! This probably doesn't have to be in Generator though, right? Has nothign to do with generation.

I would consider one of the following:
1. Put this where it is used, so outside of whole Psl module tree.
2. Put it under new file Psl.ConfigBlock.hs .
3. Put it in Psl.Ast.ConfigBlock, since it is focused on manipulating Ast.

I sorted them in the order how I like them the most, with being the most liked. Number (1) depends though on what it looks like in practice.

Reply:
Went with 3. since it felt natural to me 🤷 it's a function related to the AST